chapters/sampling-randomization-power.tex

```diff
@@ -1,427 +1,759 @@
+First, they will list all of the units who are eligible for the study,
+and enable you to map data collected or received from the field to the research design.
+Second, in designs where your team has direct control over interventions or other field work,
+they will allow you to complete sampling and treatment assignment tasks
+before those the result of those tasks is implemented in the field.
+Finally, they will function as a single unambiguous location where all information 
+related to the implementation and validity of your research is stored,
+as well as all information needed to correctly identify any observation in any of your project's datasets.
+
+The second half of this chapter covers randomization
+and the necessary practices to ensure that random processes are reproducible.
+Almost all research designs rely on a random component 
+for the results of the research to be a valid interpretation of the real world.
+This includes both how a sample is representative to the population studied,
+or how the counterfactual observations in experimental design is statistically indistinguishable
+from the treatment observations.
+The second half is concluded with a discussion on power calculations and randomization inference,
+and how both are important tools to make optimal choices when planning data work.
+
+
+\end{fullwidth}
+
+%-----------------------------------------------------------------------------------------------
+
+\section{Translating research design to master datasets}
+
+In most projects, more than one dataset is needed to answer the research question.
+This could be multiple survey rounds,
+integration of different existing data sources (e.g. administrative data,
+web scraping, implementation monitoring, etc)
+or a multitude of complex combinations of the above.
+For example, you may have different levels of unit-of-observation\sidenote{
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}},
+or the units of observation you do have (or their subunits)
+may vary from round to round.
+
+However your study is structured, you need to know how to link all the datasets you obtain
+and analyze the relationships between the units that appear in them
+to answer all your research questions.
+You might think that you are able to keep all of the relevant details in your head,
+but your whole research team is unlikely to have the same understanding,
+at all times, of all the datasets required.
+The only way to make sure that the full team shares the same understanding
+is to create \textit{master datasets} and a \textit{data map}.
+
+
+\subsection{Creating master datasets and a data map}
+
+A \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}},
+\index{master datasets}
+at the very least,
+details all project-wide time-invariant information
+about all units that could appear in the data
+and their relationship to the research design,
+like sampling and treatment status once those are determined.
```

Comment:
Suggestion from @bbdaniels : "A master dataset is created using..." I will add text on this after I have added all small edits

Reply:
@kbjarkefur is this resolved?

Reply:
done now in 71a92ade7d6d0033e88d920a5ca5db7587600898